inscriptions/ISic000547.xml

```diff
@@ -199,88 +204,23 @@
+            <div type="edition" xml:space="preserve" xml:lang="la">
                 <ab>
-                    <lb n="1"/>
-                    <expan>
-                        <abbr>D</abbr>
-                        <ex>is</ex>
-                    </expan>
-                    <expan>
-                        <abbr>M</abbr>
-                        <ex>anibus</ex>
-                    </expan>
-                    .
-                    <lb n="2"/>
-                    <expan>
-                        <abbr><supplied reason="lost">A</supplied>ur
-                        </abbr>
-                        <ex>elio</ex>
-                    </expan>
-                    Rustico
-                    <expan>
-                        <abbr>mil</abbr>
-                        <ex>itavit</ex>
-                    </expan>
-                    <lb n="3"/>
-                    <expan>
-                        <abbr>ann</abbr>
-                        <ex>os</ex>
-                    </expan>
-                    VIII
-                    <expan>
-                        <abbr>tr</abbr>
-                        <ex>iere</ex>
-                    </expan>
-                    <lb n="4"/>
-                    <g ref="#centuria">centuria</g>
-                    Zeno<lb n="5" break="no"/>nis
-                    <expan>
-                        <abbr>Mar</abbr>
-                        <ex>cus</ex>
-                    </expan>
-                    Germanici<lb n="6" break="no"/>anus
-                    <expan>
-                        <abbr>mil</abbr>
-                        <ex>itavit</ex>
-                    </expan>
-                    <expan>
-                        <abbr>ann</abbr>
-                        <ex>os</ex>
-                    </expan>
-                    X
-                    <lb n="7"/>
-                    <expan>
-                        <abbr>Valer</abbr>
-                        <ex>ius</ex>
-                    </expan>
-                    Victorinus
-                    <lb n="8"/>
-                    <expan>
-                        <abbr>her</abbr>
-                        <ex>es</ex>
-                    </expan>
-                    <expan>
-                        <abbr>b</abbr>
-                        <ex>ene</ex>
-                    </expan>
-                    <expan>
-                        <abbr>m</abbr>
-                        <ex>erenti</ex>
-                    </expan>
-                    <expan>
-                        <abbr>f</abbr>
-                        <ex>ecit</ex>
-                    </expan>
-                    .
+                    <lb n="1"/><expan><abbr>d</abbr><ex>is</ex></expan> <expan><abbr>m</abbr><ex>anibus</ex></expan>
+                    <lb n="2"/><expan><abbr><supplied reason="lost">A</supplied>ur</abbr><ex>elio</ex> <g ref="#interpunct">·</g> </expan>Rustico<expan> <g ref="#interpunct">·</g> <abbr>mil</abbr><ex>itavit</ex></expan>
```

Comment:
you need to tidy up the use of expan - the closing expan should be immediately after the closing ex after 'elio'; and the opening expan after Rustico shiould be moved to immediately before militavit?